src/js/modules/cart-template.js

```diff
@@ -0,0 +1,48 @@
+export const cartTemplate = (product) => {
```

Comment:
I dont know what is this for anyway. Everything should be wrap under a class. e.g. 
CartItemBuilder.BuildFromDataModel()